index.js

```diff
@@ -1,4 +1,10 @@
 export function scanForProblems(context, logError, options) {
+  for (const html of document.querySelectorAll('html')) {
```

Comment:
We should query on `context` not `document`. Also since `html` should be the root element, we should look upwards instead of downwards and assume there is only one `html`.